docs/products/kafka/concepts/configuration-backup.md

```diff
@@ -1,50 +1,47 @@
 ---
-title: Configuration backups for Apache Kafka®
+title: Configuration backups for Aiven for Apache Kafka®
+sidebar_label: Configuration backups
 ---
+Aiven for Apache Kafka® includes **configuration backups** that automatically back up key service configurations at no additional cost.
 
-Aiven for Apache Kafka® includes **configuration backups** that
-automatically back up key Apache Kafka® service configurations for you
-without any additional costs.
+By eliminating the need for manual reconfiguration, Aiven for Apache Kafka® configuration
+backups streamline disaster recovery and service restoration. This ensures a reliable
+way to restore your service configurations after an incident or when powering off/on your
+service, saving you valuable time and resources.
 
-Suppose the Apache Kafka® service is powered off/on or an incident
-causes the Apache Kafka cluster to stop functioning. In that case, the
-configuration backup enables the restoration of your Apache Kafka®
-service to its previous state.
+## What is backed up
 
-## Whats included in configuration backups
+Configuration backups cover the following components of your Aiven for Apache
+Kafka service:
 
-The following data are backed up in the configuration backups:
+- **Apache Kafka topics**: Including related configuration parameters such as
+  retention time and number of partitions.
+- **Apache Kafka users and ACLs**: User permissions and access control lists are included.
+- **Schema Registry data**: Including schema definitions, schema IDs, configurations
+  (such as compatibility level), subjects, and subject versions.
+- **Apache Kafka Connect configurations**: Includes all related settings.
 
--   Apache Kafka® topic-related configurations.
--   Everything related to Schema Registry (schemas, ids, subjects,
-    compatibility levels).
--   Apache Kafka® Connect configurations.
+## How backups work
 
-## Benefits of configuration backups
+- **Automatic backups**: Backups are enabled by default and stored in cloud storage
+  every 3 hours.
+- **Encryption**: Backups are encrypted for security, preventing direct access by
+  users.
+- **Automatic restoration and configuration retention**: Configuration backups are
+  automatically restored (using the latest backup) after a power off/on cycle or cluster
+  failure. This ensures your service configurations are retained and ready for use.
```

Comment:
```suggestion
  automatically restored (using the latest backup) after a power off/on cycle. This ensures your service configurations are retained and ready for use.
```

There's no automatic restoration after a generic failure. I'd leave that out.

Reply:
Thanks, @giuseppelillo! What would be the process in case of a cluster failure or similar issues? Would customers need to contact support?